src/lamp_py/bus_performance_manager/events_gtfs_rt.py

```diff
@@ -223,6 +223,10 @@ def positions_to_events(vehicle_positions: pl.DataFrame) -> pl.DataFrame:
                 "IN_TRANSIT_TO": "gtfs_travel_to_dt",
             }
         )
+        .with_columns(
```

Comment:
nit: this could be combined with the next `.with_columns()` call, my rule of thumb is to only chain a new call if a subsequent call needs to reference the new column you are creating.